Example/src/App.tsx

```diff
@@ -60,6 +60,18 @@ import { WaterfallGridExample } from './LayoutReanimation/WaterfallGridExample';
 import WobbleExample from './WobbleExample';
 import { ColorInterpolationExample } from './ColorInterpolationExample';
 import CubesExample from './CubesExample';
+import {
+  CardExample,
+  CustomTransitionExample,
+  GalleryExample,
+  LayoutAnimationExample,
+  ManyScreensExample,
+  ManyTagsExample,
+  NestedStacksExample,
+  ModalsExample,
+  FlatListExample,
+  RestoreStateExample,
+} from './SharedElementTransition';
```

Comment:
Instead of having a separate file `SharedElementTransition/index.ts` just for importing/exporting, you can simply import each example individually just like we already do.